backend/database/seed.go

```diff
@@ -200,6 +200,20 @@ func seedService(model *models.Model) {
 			CreatedAt:          time.Now(),
 			UpdatedAt:          time.Now(),
 		},
+		{
+			Name:               "Face Occlusion & Attribute",
```

Comment:
ini kalo jadi "Face Occlusion and Attribute Detection" gimana?

Reply:
di figma produk emang gitu.

Reply:
oalah yowes